css/main.css

```diff
@@ -40,15 +35,7 @@ td{
   padding: 0;
   margin: 0;
   height: 120px;
-/*  -webkit-transform: skewY(-10deg);
-  -moz-transform: skewY(-10deg); */
-  overflow: hidden;
-  /* the image is higher than the header itself */ }
-  #header img {
```

Comment:
Was this part intentionally deleted?

Reply:
Yeah, img part of header is no longer required